IlmBase/ImathTest/testExtractSHRT.cpp

```diff
@@ -67,8 +70,8 @@ testMatrix (const M33f M)
     // angle back to a matrix, N.
     //
 
-    V2f s, t;
-    float h, r;
+    V2f s(0.f), t(0.f);
```

Comment:
Since these are output variables, I wonder if there's something possible in the declaration of extractSHRT to tell the compiler it's okay that the values are uninitialized?

Reply:
well, except extractSHRT actually doesn't always initialize! if it returns false, it leaves the output values unchanged... I've added a test that it returns true here.